pygmt/src/grdview.py

```diff
@@ -117,6 +119,39 @@ def grdview(self, grid, **kwargs):
     {interpolation}
     {perspective}
     {transparency}
+
+    Example
+    -------
+    >>> import pygmt
+    >>> # load the 30 arc-minutes grid with "gridline" registration
+    >>> # in a specified region
+    >>> grid = pygmt.datasets.load_earth_relief(
+    ...     resolution="30m",
+    ...     region=[-92.5, -82.5, -3, 7],
+    ...     registration="gridline",
+    ... )
+    >>> # create a new figure instance with pygmt.Figure()
+    >>> fig = pygmt.Figure()
+    >>> # create the contour plot
+    >>> fig.grdview(
+    ...     # pass in the grid downloaded above
+    ...     grid=grid,
+    ...     # set the perspective to an azimuth of 130° and an elevation of 30°
+    ...     perspective=[130, 30],
+    ...     # add a frame to the x- and y-axes
+    ...     # specify annotations on the south and east borders of the plot
+    ...     frame=["xa", "ya", "wSnE"],
+    ...     # set the projection to Mercator for the 10 cm wide figure
+    ...     projection="M10c",
```

Comment:
Hm. Unforantly I feel my review in https://github.com/GenericMappingTools/pygmt/pull/2381#discussion_r1141282033 was not precise 🙁. This is a 3-D figure and width and hight of the figure change when changing the `perspective` argument. Probably this width refers to the map or plot itself? 
```suggestion
    ...     # set the projection to Mercator for the 10 cm wide plot
    ...     projection="M10c",
```

Reply:
`M10c` means the frame width is 10-cm in 2D view. Thus, in 3D view, the actually width can be any other values.

Reply:
Ah, OK. This makes sense.

Reply:
See my comment https://github.com/GenericMappingTools/pygmt/pull/2381/files#r1141549661.